subprojects/docs/build.gradle

```diff
@@ -535,6 +535,13 @@ samples {
             description = "Developing a Gradle plugin in a build without publishing."
             category = "Build organization"
         }
+
+        crossProjectOutputSharing {
+            sampleDirectory = samplesRoot.dir("build-organization/cross-project-output-sharing")
+            displayName = "Sharing task outputs across projects in a multi-project build"
+            description = "Sharing a file made by a task in one Gradle project, with a task in another Gradle project."
+            category = "Build organization"
```

Comment:
I am not sure if this belongs to build organization. Although I am not sure where it should belong. Is there a good reason it's in build organization?

Reply:
Yeah, I'm not completely sure either. I put it there because it's specifically something you use when working with multi-project builds and you're organising producers & consumers of files across your build logic.

The current categories are:

<img width="190" alt="Screenshot 2022-01-18 at 9 57 33 AM" src="https://user-images.githubusercontent.com/1451027/149857494-ac5850ed-c76f-4a00-b705-176a43e173f6.png">

I think build organisation is the best fit out of these, but I don't know. What do you think?

Reply:
Works for me